### DISCUSSION TITLE
Multiple companies, one Kimai

### DISCUSSION THREAD

**Discussion by jfbethlehem** [kimai/kimai#4736] [General]
Is it possible to have multiple companies using the same Kimai instance? For example, I have a Kimai instance where I want to run ParComp and ChildComp at the same time. I will be putting hours under ParComp and employees under ChildComp. This due to tax and financial security issues.
I would love to then be able to make invoices logged in to ParComp for ChildComp and download those invoices to send out to the various clients ChildComp has. Is this possible? Would it need some kind of extension?

**Comment by kevinpapst** [comment]
Yes, why not... you just have to carefully craft your team setup, to prevent that data leaks from one company into the other.

**Reply by jfbethlehem** [reply]
How should the teams be crafted?

**Reply by kevinpapst** [reply]
At least two teams, one for each company. Then each customer needs to be assigned to either none or one team.

1. Read the docs
2. Test
3. Come back with detail questions
